LeWricka/PrAnalytics
# [4] - Send comments from PR sent to elasticsearch

Resolves #4 

Elasticsearch docker configuration updated and controller basic logic implemented to send comments data to elasticseach.

How to test:
- Open ngrok port.
- Add endpoint on github config.
- Create pr.
- Add comments.
- Check elasticsearch or kibana
